Invariant Violation: addComponentAsRefTo(...): Only a ReactOwner can have refs

Related to #13, #9 and #14 

I installed as per the instructions and performed the `apm install`. I also then `npm i underscore react react-dom` which gave me:

```
underscore@1.8.3 node_modules/underscore

react-dom@15.1.0 node_modules/react-dom

react@15.1.0 node_modules/react
├── object-assign@4.1.0
├── loose-envify@1.2.0 (js-tokens@1.0.3)
└── fbjs@0.8.2 (ua-parser-js@0.7.10, promise@7.1.1, core-js@1.2.6, isomorphic-fetch@2.2.1)
```

And it does not load the Markdown editor.

Please excuse the masshoosive stack traces :)

## Log from launching normally:

```js
/Applications/Nylas N1.app/Contents/Resources/app.asar/src/nylas-env.js:306 Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.
    at invariant (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/node_modules/fbjs/lib/inv…:34)
    at Object.ReactOwner.removeComponentAsRefFrom (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactOwner.js:81)
    at detachRef (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactRef.js:32)
    at Object.ReactRef.detachRefs (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactRef.js:74)
    at Object.ReactReconciler.unmountComponent (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconciler.js:64)
    at Object.ReactChildReconciler.unmountChildren (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactChildReconciler.…:118)
    at ReactDOMComponent.ReactMultiChild.Mixin.unmountChildren (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMultiChild.js:325)
    at ReactDOMComponent.Mixin.unmountComponent (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDOMComponent.js:887)
    at Object.ReactReconciler.unmountComponent (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconciler.js:65)
    at ReactCompositeComponentMixin.unmountComponent (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactCompositeCompone…:329)
    at Object.ReactReconciler.unmountComponent (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconciler.js:65)
    at ReactCompositeComponentMixin.unmountComponent (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactCompositeCompone…:329)
    at Object.ReactReconciler.unmountComponent (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconciler.js:65)
    at unmountComponentFromNode (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:138)
    at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDefaultBatchingS…:61)
    at Object.batchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:97)
    at Object.ReactMount.unmountComponentAtNode (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:409)
    at Object.ReactMount._renderSubtreeIntoContainer (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:330)
    at Object.ReactMount.render (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:374)
    at UnsafeComponent.renderInjected (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:128)
    at UnsafeComponent.renderInjected (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:3)
    at UnsafeComponent.componentDidMount (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:76)
    at UnsafeComponent.componentDidMount (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:3)
    at CallbackQueue._assign.notifyAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/CallbackQueue.js:66)
    at ReactReconcileTransaction.ON_DOM_READY_QUEUEING.close (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconcileTransac…:79)
    at ReactReconcileTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202)
    at ReactReconcileTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149)
    at ReactUpdatesFlushTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:136)
    at ReactUpdatesFlushTransaction._assign.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:89)
    at Object.flushBatchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:165)
    at ReactDefaultBatchingStrategyTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202)
    at ReactDefaultBatchingStrategyTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149)
    at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDefaultBatchingS…:63)
    at Object.enqueueUpdate (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:194)
    at enqueueUpdate (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:22)
    at Object.ReactUpdateQueue.enqueueSetState (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:201)
    at ReactComponent.setState (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactComponent.js:67)
    at _nylasExports.DraftStore.sessionForClientId.then.session (inflates-draft-client-id.es6:61)
    at tryCatcher (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/util.js:26)
    at Promise._settlePromiseFromHandler (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:507)
    at Promise._settlePromiseAt (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:581)
    at Promise._settlePromiseAtPostResolution (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:245)
    at Async._drainQueue (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:128)
    at Async._drainQueues (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:133)
    at Immediate.Async.drainQueues [as _onImmediate] (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:15)
    at tryOnImmediate (timers.js:534)
    at processImmediate [as _immediateCallback] (timers.js:514)
```

## Log from running with debug flags:

```js
/Applications/Nylas N1.app/Contents/Resources/app.asar/src/nylas-env.js:306 Invariant Violation: addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a components `render` method, or you have multiple copies of React loaded (details: https://fb.me/react-refs-must-have-owner).
    at invariant (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/node_modules/fbjs/lib/invariant.js:38:15)
    at Object.ReactOwner.addComponentAsRefTo (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactOwner.js:67:79)
    at attachRef (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactRef.js:23:16)
    at Object.ReactRef.attachRefs (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactRef.js:42:5)
    at ReactDOMComponent.attachRefs (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconciler.js:22:12)
    at CallbackQueue._assign.notifyAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/CallbackQueue.js:66:22)
    at ReactReconcileTransaction.ON_DOM_READY_QUEUEING.close (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconcileTransaction.js:79:26)
    at ReactReconcileTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202:25)
    at ReactReconcileTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149:16)
    at batchedMountComponentIntoNode (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:124:15)
    at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDefaultBatchingStrategy.js:61:7)
    at Object.batchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:97:20)
    at Object.ReactMount._renderNewRootComponent (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:277:18)
    at Object.wrapper [as _renderNewRootComponent] (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactPerf.js:66:21)
    at Object.ReactMount._renderSubtreeIntoContainer (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:354:32)
    at Object.ReactMount.render (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMount.js:374:23)
    at Object.wrapper [as render] (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactPerf.js:66:21)
    at UnsafeComponent.renderInjected (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:109:41)
    at UnsafeComponent.renderInjected (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:3:61)
    at UnsafeComponent.componentDidMount (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:76:19)
    at UnsafeComponent.componentDidMount (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:3:61)
    at CallbackQueue._assign.notifyAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/CallbackQueue.js:66:22)
    at ReactReconcileTransaction.ON_DOM_READY_QUEUEING.close (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconcileTransaction.js:79:26)
    at ReactReconcileTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202:25)
    at ReactReconcileTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149:16)
    at ReactUpdatesFlushTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:136:20)
    at ReactUpdatesFlushTransaction._assign.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:89:38)
    at Object.flushBatchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:165:19)
    at Object.wrapper [as flushBatchedUpdates] (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactPerf.js:66:21)
    at ReactDefaultBatchingStrategyTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202:25)
    at ReactDefaultBatchingStrategyTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149:16)
    at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDefaultBatchingStrategy.js:63:19)
    at Object.enqueueUpdate (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:194:22)
    at enqueueUpdate (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:22:16)
    at Object.ReactUpdateQueue.enqueueSetState (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:201:5)
    at ReactComponent.setState (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactComponent.js:67:16)
    at _nylasExports.DraftStore.sessionForClientId.then.session (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/composer/lib/decorators/inflates-draft-client-id.js:64:14)
    at tryCatcher (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:507:31)
    at Promise._settlePromiseAt (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:581:18)
    at Promise._settlePromiseAtPostResolution (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:245:10)
    at Async._drainQueue (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:128:12)
    at Async._drainQueues (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:15:14)
    at tryOnImmediate (timers.js:534:15)
    at processImmediate [as _immediateCallback] (timers.js:514:5)module.exports.NylasEnvConstructor.reportError @ /Applications/Nylas N1.app/Contents/Resources/app.asar/src/nylas-env.js:306UnsafeComponent.renderInjected @ /Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:130(anonymous function) @ /Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:3UnsafeComponent.componentDidMount @ /Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:76(anonymous function) @ /Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/unsafe-component.js:3_assign.notifyAll @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/CallbackQueue.js:66ON_DOM_READY_QUEUEING.close @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconcileTransac…:79Mixin.closeAll @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202Mixin.perform @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149Mixin.perform @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:136_assign.perform @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:89flushBatchedUpdates @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:165ReactUpdates_flushBatchedUpdates @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactPerf.js:66Mixin.closeAll @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202Mixin.perform @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149ReactDefaultBatchingStrategy.batchedUpdates @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDefaultBatchingS…:63enqueueUpdate @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:194enqueueUpdate @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:22ReactUpdateQueue.enqueueSetState @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:201ReactComponent.setState @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactComponent.js:67_nylasExports.DraftStore.sessionForClientId.then.session @ inflates-draft-client-id.es6:61tryCatcher @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/util.js:26Promise._settlePromiseFromHandler @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:507Promise._settlePromiseAt @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:581Promise._settlePromiseAtPostResolution @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/promise.js:245Async._drainQueue @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:128Async._drainQueues @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:133Async.drainQueues @ /Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/bluebird/js/main/async.js:15tryOnImmediate @ timers.js:534processImmediate @ timers.js:514

/Applications/Nylas N1.app/Contents/Resources/app.asar/src/nylas-env.js:306 TypeError: Cannot read property 'getCurrentSelection' of undefined
    at InjectedComponent._runInnerDOMMethod (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/injected-component.js:197:82)
    at InjectedComponent._runInnerDOMMethod (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/injected-component.js:3:61)
    at InjectedComponent.Object.defineProperty.value [as getCurrentSelection] (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/components/injected-component.js:218:47)
    at ComposerView._getSelections (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/composer/lib/composer-view.js:304:77)
    at ComposerView._saveToHistory.selections [as _saveToHistory] (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/composer/lib/composer-view.js:310:37)
    at ComposerView._receivedNewSession (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/composer/lib/composer-view.js:403:10)
    at ComposerView.componentDidMount (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/composer/lib/composer-view.js:343:12)
    at CallbackQueue._assign.notifyAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/CallbackQueue.js:66:22)
    at ReactReconcileTransaction.ON_DOM_READY_QUEUEING.close (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconcileTransaction.js:79:26)
    at ReactReconcileTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202:25)
    at ReactReconcileTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149:16)
    at ReactUpdatesFlushTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:136:20)
    at ReactUpdatesFlushTransaction._assign.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:89:38)
    at Object.flushBatchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:165:19)
    at Object.wrapper [as flushBatchedUpdates] (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactPerf.js:66:21)
    at ReactDefaultBatchingStrategyTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202:25)
    at ReactDefaultBatchingStrategyTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149:16)
    at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDefaultBatchingStrategy.js:63:19)
    at Object.enqueueUpdate (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:194:22)
    at enqueueUpdate (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:22:16)
    at Object.ReactUpdateQueue.enqueueSetState (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:201:5)
    at ReactComponent.setState (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactComponent.js:67:16)
    at _nylasExports.DraftStore.sessionForClientId.then.session (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/composer/lib/decorators/inflates-draft-client-id.js:64:14)
    at tryOnImmediate (timers.js:534:15)
    at processImmediate [as _immediateCallback] (timers.js:514:5)
From previous event:
    at _prepareForDraft (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/composer/lib/decorators/inflates-draft-client-id.js:52:66)
    at componentDidMount (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/composer/lib/decorators/inflates-draft-client-id.js:32:12)
    at CallbackQueue._assign.notifyAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/CallbackQueue.js:66:22)
    at ReactReconcileTransaction.ON_DOM_READY_QUEUEING.close (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconcileTransaction.js:79:26)
    at ReactReconcileTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202:25)
    at ReactReconcileTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149:16)
    at ReactUpdatesFlushTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:136:20)
    at ReactUpdatesFlushTransaction._assign.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:89:38)
    at Object.flushBatchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:165:19)
    at Object.wrapper [as flushBatchedUpdates] (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactPerf.js:66:21)
    at ReactDefaultBatchingStrategyTransaction.Mixin.closeAll (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:202:25)
    at ReactDefaultBatchingStrategyTransaction.Mixin.perform (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/Transaction.js:149:16)
    at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDefaultBatchingStrategy.js:63:19)
    at Object.enqueueUpdate (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdates.js:194:22)
    at enqueueUpdate (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:22:16)
    at Object.ReactUpdateQueue.enqueueSetState (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactUpdateQueue.js:201:5)
    at MessageList.ReactComponent.setState (/Applications/Nylas N1.app/Contents/Resources/app.asar/node_modules/react/lib/ReactComponent.js:67:16)
    at MessageList._onChange (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/message-list/lib/message-list.js:568:19)
    at MessageStore.<anonymous> (/Applications/Nylas N1.app/Contents/Resources/app.asar/internal_packages/message-list/lib/message-list.js:3:61)
    at EventEmitter.eventHandler (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/flux/modules/reflux-coffee.js:206:27)
    at emitOne (events.js:95:20)
    at EventEmitter.emit (events.js:182:7)
    at MessageStore.module.exports.Publisher.trigger (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/flux/modules/reflux-coffee.js:218:30)
    at MessageStore._onDataChanged (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/flux/stores/message-store.js:185:18)
    at MessageStore._onDataChanged (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/flux/stores/message-store.js:3:61)
    at EventEmitter.eventHandler (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/flux/modules/reflux-coffee.js:206:27)
    at emitOne (events.js:95:20)
    at EventEmitter.emit (events.js:182:7)
    at DatabaseStore.module.exports.Publisher.trigger (/Applications/Nylas N1.app/Contents/Resources/app.asar/src/flux/modules/reflux-coffee.js:218:30)
    at /Applications/Nylas N1.app/Contents/Resources/app.asar/src/flux/stores/database-store.js:639:17
```

--- a/lib/markdown-editor.cjsx
+++ b/lib/markdown-editor.cjsx
@@ -1,8 +1,9 @@
 _ = require 'underscore'
 Utils = require './utils'
 SimpleMDE = require 'simplemde'
-React = require 'react'
-ReactDOM = require 'react-dom'
+nylas = require 'nylas-exports'
+React = nylas.React
+ReactDOM = nylas.ReactDOM
 
 # Keep a file-scope variable containing the contents of the markdown stylesheet.
 # This will be embedded in the markdown preview iFrame, as well as the email body.

--- a/package.json
+++ b/package.json
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "simplemde": "jstejada/simplemde-markdown-editor",
-    "marked": "^0.3"
+    "marked": "^0.3",
+    "underscore": "^1.8"
   }
 }

